fmriprep/workflows/bold/stc.py

```diff
@@ -93,22 +93,29 @@ def init_bold_stc_wf(metadata, name='bold_stc_wf'):
     from niworkflows.engine.workflows import LiterateWorkflow as Workflow
     from niworkflows.interfaces.header import CopyXForm
 
+    slice_times = metadata["SliceTiming"]
+    first, last = min(slice_times), max(slice_times)
+    frac = config.workflow.slice_time_ref
+    tzero = first + frac * (last - first)
```

Comment:
Does it make sense to explicitly round to millisecond resolution?

```suggestion
    tzero = np.round(first + frac * (last - first), 3)
```

Reply:
Oh, @mgxd forgot to ask about this.

Reply:
it looks like afni's tshift assigns this as a [float datatype](https://github.com/afni/afni/blob/25e77d564f2c67ff480fa99a7b8e48ec2d9a89fc/src/3dTshift.c#L201), which supports up to 6 decimal digits of precision. though that may be excessive

Reply:
True. I'm just imagining that any extra significant digits are rounding errors from performing this calculation. And we report it to `{:.03g}`, so this would ensure that what is sent to AFNI and reported to the user are the same.

Reply:
I have no strong opinion, fine with rounding to millisecond